### ISSUE TITLE
Add tooltips to talks list

### ISSUE THREAD

**Issue by nicogaldamez** [adrienpoly/rubyvideo#405]
<img width="1221" alt="image" src="https://github.com/user-attachments/assets/eac7dc8d-c74c-414c-8afd-b2bd3314d5be">

**Comment by marcoroth** [comment]
Agreed. On a related note, I think we should use something like Tippy.js to have proper positioning and flexibility. The daisyUI built-in tooltips are really bad and only work on <button> tags.

**Comment by nicogaldamez** [comment]
Resolved via #425
